scripts/setup/Dockerfile

```diff
@@ -43,26 +51,30 @@ RUN apt-get update && \
     libvirt-clients \
     libvirt-daemon-system \
     mtools \
+    netcat-openbsd \
     ninja-build \
     pkg-config \
     python3-venv \
     qemu-kvm \
     qemu-system-x86 \
     texinfo \
-    xorriso && \
+    xorriso \
+    wget && \
     apt-get clean && \
     rm -rf /var/lib/apt/lists/*
 
 # Install Rust.
 RUN curl https://sh.rustup.rs -sSf | sh -s -- -y
 ENV PATH=$HOME/.cargo/bin:$PATH
-RUN rustup toolchain install $RUST_VERSION
-RUN rustup default $RUST_VERSION
-RUN rustup component add rust-src
-RUN rustup target add wasm32-wasip1
 
-# Build toolchain.
-RUN git clone https://github.com/nanvix/nanvix /opt/src/nanvix
-RUN cd /opt/src/nanvix && git checkout -f $NANVIX_VERSION
-RUN cd /opt/src/nanvix && scripts/setup/toolchain.sh /opt
-RUN rm -rf /opt/nanvix/src
+# Install Rust toolchain and components.
+RUN rustup toolchain install $RUST_VERSION && \
+    rustup default $RUST_VERSION && \
+    rustup component add rust-src && \
+    rustup target add wasm32-wasip1
+
+# Clone Nanvix repository and build toolchain.
+RUN git clone https://github.com/nanvix/nanvix /opt/nanvix/src/nanvix && \
+    cd /opt/nanvix/src/nanvix && \
+    git checkout -f $NANVIX_VERSION && \
+    scripts/setup/toolchain.sh /opt/nanvix
```

Comment:
Just double-checking. From what I remember `scripts/setup/toolchain.sh` also required `TOOLCHAIN_DIR` being set as an env. variable? Maybe not anymore.